pkg/components/helpers.go

```diff
@@ -300,3 +305,104 @@ func getDNSConfigWithDefault(componentDNSConfig, defaultDNSConfig *corev1.PodDNS
 	// Otherwise, fall back to the default DNSConfig.
 	return defaultDNSConfig
 }
+
+func getTimbertruckInitScript(timbertruckConfig *ytconfig.TimbertruckConfig) (string, error) {
+	configText, err := yaml.Marshal(timbertruckConfig)
+	if err != nil {
+		return "", fmt.Errorf("failed to marshal Timbertruck config: %w", err)
+	}
+	return fmt.Sprintf("%s%s%s", timbertruckInitScriptPrefix, string(configText), timbertruckInitScriptSuffix), nil
+}
+
+func prepareTimbertruckTablesFromConfig(ctx context.Context, ytClient yt.Client, timbertruckConfig *ytconfig.TimbertruckConfig) error {
+	for _, jsonLog := range timbertruckConfig.JsonLogs {
+		for _, ytQueue := range jsonLog.YTQueue {
+			queueExportPath := fmt.Sprintf("//sys/admin/logs/export/%s", jsonLog.Name)
```

Comment:
All of this needs to be configurable. Paths, export timings, etc. We have some similar configuration with delivery via dynamic table log writer, and we will want to keep the same structure (and delivering logs twice is useless). 

Also, people will want to have access logs not only for access log viewer tab, but also to run YQL analysis on the raw access log (for audit events/leaks, for example), and 15 minute exports are not convenient for that. 

So it is probably best to have the ability to configure multiple exports, shorter 15 minute ones for access log viewer, and daily/weekly ones for historical usage.

Reply:
Can you share our production configuration for the reference? Both of the queue's exports and the dynamic table log writer.